lib/order.rb

```diff
@@ -0,0 +1,53 @@
+class Order
+
+  attr_reader :order_as_hash, :menu, :actual_total, :customer_supplied_total, :text
```

Comment:
Metrics/LineLength: Line is too long. [83/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)